index.html

```diff
@@ -13,7 +14,85 @@
   <![endif]-->
 </head>
 
-<body>
-  <h1>My Resume</h1>
-</body>
+<div class="container">
+
+  <body>
+    <nav>
+      <nav class="navbar">
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <a href="#">Portfolio</a>
+        <a href="#">Contact</a>
+      </nav>
+
+      <section class="container cta">
+        <div class="cta-top">
```

Comment:
I noticed that all of your divs have the container class. Is that intentional? It looks like you ended up needing to manually override some CSS as a result. The better approach (using LESS), would be to create a mixin with the:
margin: 0 auto;
width: 100%;
max-width: 900px;
if that's something you want, and then apply the mixin where you want.